### ISSUE TITLE
No possibility to accept self signed certificate

### ISSUE THREAD

**Issue by VMbuseck** [bhdouglass/owncloud-files-scope#5]
It isn't possible to work with self signed certificate.

**Comment by ernesst** [comment]
Maybe you can download it and manually install it ?
Otherwise there is the solution of Letsencrypt

**Comment by bhdouglass** [comment]
As @ernesst, I recommend changing to a real ssl certificate from https://letsencrypt.org/
I may be able to provide an option to ignore those errors.

**Comment by VMbuseck** [comment]
Letsencrypt is no option at my ISP. You must buy a certificate for at least 1,99 € a month.

**Comment by bhdouglass** [comment]
Well that's annoying! I'll look into an option to ignore the SSL errors.

**Comment by laryllan78** [comment]
@bhdouglass 

I filed a bug against web-browser-app which obviously can't cope with self-signed certificates:
https://bugs.launchpad.net/ubuntu/+source/webbrowser-app/+bug/1523261

This bug is known, but it seems nobody cares about it right now:
https://bugs.launchpad.net/oxide/+bug/1503035

I hope you can make it work.

At the moment, accessing a private Owncloud with a self-signed certificate is not possible on Ubuntu for phones.
Let's Encrypt is no solution, because it doesn't sign DynDNS providers in general.

**Comment by VMbuseck** [comment]
> Let's Encrypt is no solution, because it doesn't sign DynDNS providers in general.

That's not true. I have signed 6 domains DynDNS domains with Let's Encrypt.

**Comment by bhdouglass** [comment]
I believe this can be done with a CA setup, but it wouldn't be user friendly at all. Let's Encrypt is a preferred solution.

**Comment by ernesst** [comment]
I think you can manually install self signed certificat by ourself and the connect via the browser

**Comment by laryllan78** [comment]
> That's not true. I have signed 6 DynDNS domains with Let's Encrypt.

Sure, but it doesn't work with other providers. I cannot use DynDNS.

> I think you can manually install self signed certificat by ourself and the connect via the browser

No you can't. The browser cannot access the Owncloud web frontend with a self-signed certificate.
This it what the launchpad bugs posted above are about.

**Comment by ernesst** [comment]
According to http://askubuntu.com/questions/591866/how-can-contacts-and-calendars-be-synchronised/600629#600629

```
sudo mount /dev/loop0 / -o remount,rw
sudo cp /home/phablet/Downloads/server.crt /usr/share/ca-certificates
sudo dpkg-reconfigure ca-certificates
```

**Comment by VMbuseck** [comment]
This will solve the issue with a 'real' selfsigned certificate, but not generic certificates deployed by the ISP in a shared host environment, because ISP-Certificate does not match with the hostname.

For example: Domain factory deploys a certificate for 'webserver.ispgateway.de' if I call https://cloud.mysite.de

**Comment by bhdouglass** [comment]
I've just released v1.8 which has a new option to ignore self signed certificate errors. Let me know if that works for you guys!
